include/zrepl_prot.h

```diff
@@ -169,6 +169,7 @@ enum zvol_status {
 
 typedef enum zvol_status zvol_status_t;
 
+#define	ZVOL_IS_HEALTHY(zv) (zv->zv_status == ZVOL_STATUS_HEALTHY)
```

Comment:
lets us uzfs_zinfo_get_status(zinfo) getter function

Reply:
There are some more macros which are also not as per guidelines. 
#define ZINFO_IS_DEGRADED(zinfo)        (ZVOL_IS_DEGRADED(zinfo->main_zv))
#define ZVOL_IS_DEGRADED(zv)            (zv->zv_status == ZVOL_STATUS_DEGRADED)
Let me fix them with next PR. This PR seems to be little heavy weight.